packages/go/openapi/src/paths/attack-paths.domains.id.details.yaml

```diff
@@ -25,33 +25,34 @@ parameters:
 get:
   operationId: ListDomainAttackPathsDetails
   summary: List domain attack paths details
-  description: Lists detailed data about attack paths for a domain.
+  description: |
+    Lists detailed data about attack paths for a domain. 
+    
+    __Note:__ __Note:__ `ImpactCount`, `ImpactPercentage`, `ExposureCount`, and `ExposurePercentage` will have a value other than zero when butterfly analysis is enabled.

```

Comment:
We should also add a blurb here about how `severity` will be empty when the feature flag is disabled.

Reply:
updated